Geany 1.33

Mno-hime opened the pull request:
Run tested, plugins load and activate.

alarcher commented:
You need to split in two PRs: first geany, then geany-plugins as the latter cannot build against geany 1.31. Thanks!

Mno-hime committed:
Geany 1.33

Run tested, plugins load and activate.

Plugins: https://github.com/OpenIndiana/oi-userland/pull/4174

Mno-hime commented:
Geany plugins 1.33: https://github.com/OpenIndiana/oi-userland/pull/4174

alarcher commented:
Can you add the lv locale to the manifest?

alarcher commented on components/editor/geany/manifests/sample-manifest.p5m:
```diff
@@ -263,6 +263,7 @@ file path=usr/share/locale/kk/LC_MESSAGES/geany.mo
 file path=usr/share/locale/ko/LC_MESSAGES/geany.mo
 file path=usr/share/locale/lb/LC_MESSAGES/geany.mo
 file path=usr/share/locale/lt/LC_MESSAGES/geany.mo
+file path=usr/share/locale/lv/LC_MESSAGES/geany.mo
```
This should be added to geany.p5m as well.
